Mathlib/RingTheory/Congruence/Basic.lean

```diff
@@ -75,8 +75,8 @@ instance : FunLike (RingCon R) R (R → Prop) :=
       rcases x with ⟨⟨x, _⟩, _⟩
       rcases y with ⟨⟨y, _⟩, _⟩
       congr!
-      rw [Setoid.ext_iff,(show x.Rel = y.Rel from h)]
-      simp}
+      rw [Setoid.ext_iff, (show x.Rel = y.Rel from h)]
+      simp }
```

Comment:
While you're here, can you convert this instance to use `where` so that the `}` can be dropped?

Reply:
And please remove the porting note on L70, because that seems to have been done.